.github/workflows/checks.yaml

```diff
@@ -0,0 +1,58 @@
+name: Checks
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  ci: 
+    runs-on: ubuntu-latest
+    outputs:
+      status: ${{ job.status }}
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name : Run EC Validate (keyless)
+      uses: ./
+      with:
+        image: ghcr.io/enterprise-contract/golden-container:latest
+        identity: https:\/\/github\.com\/(slsa-framework\/slsa-github-generator|enterprise-contract\/golden-container)\/
+        issuer: https://token.actions.githubusercontent.com
+
+    # - name : Run EC Validate (Long_Lived)
+    #   uses: ./
+    #   with:
+    #     image: quay.io/redhat-appstudio/ec-golden-image:latest
+    #     key: ${{ vars.PUBLIC_KEY }}
+    #     policy: " " #TODO Ignore until image is fixed 
+    #     extra-params: --ignore-rekor
```

Comment:
Let's explain more clearly in comments why this is commented out.

Reply:
Done hopefully more clear now 👍🏻

Reply:
I wonder if this change fixed it: https://github.com/enterprise-contract/golden-container/pull/129

As a side note. Usually, a comment like this would go as the first line of the commented out block so it is obvious to the reader, e.g.:

```
    # FIXME Commented out because ...
    # - name : Run EC Validate (Long_Lived)
    #   uses: ./
    #   with:
```

Reply:
It is now a warning I am going try fix it today 
```
 warnings:
  - metadata:
      code: slsa_source_correlated.source_code_reference_provided
    msg: Expected source code reference was not provided for verification
```